.github/workflows/ci.yml

```diff
@@ -167,17 +167,24 @@ jobs:
           brew install gnu-tar
           echo "/usr/local/opt/gnu-tar/libexec/gnubin" >> $GITHUB_PATH
 
-      - name: Cache
+      - name: Cache Cargo home
+        uses: actions/cache@v2
+        with:
+          # See https://doc.rust-lang.org/cargo/guide/cargo-home.html#caching-the-cargo-home-in-ci
+          path: |
+            ~/.cargo/registry/index
+            ~/.cargo/registry/cache
+            ~/.cargo/git/db
+          key:
+            a-cargo-home-${{ hashFiles('Cargo.lock') }}
```

Comment:
I think cargo home isn't platform-dependent. So I didn't include `matrix.os` in the cache key.